### ISSUE TITLE
Security vulnerability: marked package

### ISSUE THREAD

**Issue by yciurlik-sage** [jsdoc2md/jsdoc-to-markdown#231]
Hi,

There is a security issue on the **marked** dependency on jsdoc-to-markdown v6.0.1.

```

                       === npm audit security report ===                        


                                 Manual Review                                  
             Some vulnerabilities require your attention to resolve             
                                                                                
          Visit https://go.npm.me/audit-guide for additional guidance           


  Moderate        Regular Expression Denial of Service                          

  Package         marked

  Patched in      >=2.0.0

  Dependency of   jsdoc-to-markdown [dev]

  Path            jsdoc-to-markdown > dmd > marked

  More info       https://npmjs.com/advisories/1623

found 1 moderate severity vulnerability in 476 scanned packages
  1 vulnerability requires manual review. See the full report for details.
```

**Comment by 75lb** [comment]
jsdoc2md is an offline command-line tool not an online service permentantly exposed to the internet and hackers.. why is this an issue?

**Comment by 75lb** [comment]
Fixed and released in v7.0.0 - all dependencies refreshed.
